doc/cephfs/mount-using-fuse.rst

```diff
@@ -2,24 +2,27 @@
  Mount CephFS using FUSE
 ========================
 
-`ceph-fuse`_ is an alternate way of mounting CephFS, although it mounts it
-in userspace. Therefore, performance of FUSE can be relatively lower but FUSE
-clients can be more manageable, especially while upgrading CephFS.
+`ceph-fuse`_ can be used as an alternative to the :ref:`CephFS kernel
+driver<cephfs-mount-using-kernel-driver>` to mount CephFS. `ceph-fuse`_ mounts
+are made in userspace. This means that `ceph-fuse`_ mounts are less performant
+than kernel driver mounts, but they are easier to manage and easier to upgrade.
 
 Prerequisites
 =============
 
-Go through the prerequisites required by both, kernel as well as FUSE mounts,
-in `Mount CephFS: Prerequisites`_ page.
+Ensure that you have all the prerequisites required by both kernel and FUSE
+mounts, as listed on the `Mount CephFS: Prerequisites`_ page.
 
-.. note:: Mounting CephFS using FUSE requires superuser privileges to trim dentries
-   by issuing a remount of itself.
+.. note:: Mounting CephFS using FUSE requires superuser privileges to trim
+   dentries by issuing a remount of itself.
```

Comment:
😵

Reply:
Yeah, I've already written to Patrick on this matter.

Reply:
I get what it’s saying, but don’t have a suggestion to make it less elliptical.

Reply:
Patrick rewrote this for us in https://github.com/ceph/ceph/pull/58596.